echo_handlers.go

```diff
@@ -309,6 +313,28 @@ func (svc *Service) AppsCreateHandler(c echo.Context) error {
 			if err != nil {
 				return err
 			}
+
+			requestMethod := c.FormValue("RequestMethod")
+			if requestMethod != "" {
+				//validate requestMethod
+				//should be space seperated, and we always create the pay_invoice permission anyway
+				//only create the get_balance if present now
+				methodsToCreate := strings.Split(requestMethod, " ")
```

Comment:
This code will only execute if maxAmount is set or expiresAt is set.

It would be cool to be able to make a connection that can just read your balance without being able to make payments for example. In this case this code might need to be changed a bit.

Reply:
I can't really think of anything that would use this interface to _only_ read your balance to be honest..